cpp/tests/large_strings/json_tests.cu

```diff
@@ -15,8 +15,8 @@
  */
 
 #include "../io/json/json_utils.cuh"
-#include "io/comp/comp.hpp"
-#include "io/comp/io_uncomp.hpp"
+#include "io/comp/compression.hpp"
```

Comment:
Instead of these, shouldn't we just now need to include `#include <cudf/io/codec.hpp>`